Kudu.Contracts/Settings/SettingsKeys.cs

```diff
@@ -34,5 +34,6 @@ public static class SettingsKeys
         public const string DisableSubmodules = "SCM_DISABLE_SUBMODULES";
         public const string SiteExtensionsFeedUrl = "SCM_SITEEXTENSIONS_FEED_URL";
         public const string DisableDeploymentOnPush = "SCM_DISABLE_DEPLOY_ON_PUSH";
+        public const string AlwaysTouchWebConfigAfterDeployment = "SCM_ALWAYS_TOUCH_WEBCONGIF_AFTER_DEPLOYMENT";
```

Comment:
Typo: `CONGIF`. Also, I would yank the ALWAYS. So `SCM_TOUCH_WEBCONFIG_AFTER_DEPLOYMENT`.